### PR TITLE
fix and add interface of Permission Precompiled Contract

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/permission/permission_service_test.go  3
- precompiled/config/system_config_service.go  1
- precompiled/permission/permission_service.go  4
         

Clones added
============
- precompiled/permission/permission_service_test.go  3
- precompiled/config/system_config_service.go  1
- precompiled/permission/permission_service.go  4
- precompiled/permission/permission.go  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5870056&bid=19214127)

File: precompiled/permission/permission_service.go [hunk 1]
```diff
@@ -1,11 +1,12 @@
 package permission
 
 import (
-	"context"
-	"crypto/ecdsa"
 	"encoding/json"
 	"fmt"
-	"math/big"
+
+	"github.com/FISCO-BCOS/go-sdk/precompiled"
```
Comment (bxq2011hust) [inline precompiled/permission/permission_service.go:7]: format

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/permission/permission_service.go  4
- precompiled/config/system_config_service.go  1
- precompiled/permission/permission_service_test.go  3
         

Clones added
============
- precompiled/permission/permission_service.go  4
- precompiled/config/system_config_service.go  1
- precompiled/permission/permission_service_test.go  3
- precompiled/permission/permission.go  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5870056&bid=19214127)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/permission/permission_service.go  4
- precompiled/permission/permission_service_test.go  3
- precompiled/config/system_config_service.go  1
         

Clones added
============
- precompiled/permission/permission_service.go  4
- precompiled/permission/permission_service_test.go  3
- precompiled/config/system_config_service.go  1
- precompiled/permission/permission.go  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5870056&bid=19214127)

**Pull request merged**